code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -960,6 +960,13 @@
 	junkiness = 25
 	list_reagents = list("nutriment" = 1, "beans" = 3, "msg" = 4, "sugar" = 2)
 
+/obj/item/weapon/reagent_containers/food/snacks/chinese/sweetsourchickenball
+	name = "Sweet & Sour Chicken Balls"
+	desc = "Is this chicken cooked? The odds are better than wok paper scissors."
+	icon_state = "chickenball"
+	junkiness = 25
+	list_reagents = list("nutriment" = 2, "msg" = 4, "sugar" = 2)
```

Comment:
Leaning towards saying this should be protein not bland nutriment... but it's Mr. Changs so who knows what is actually in there.

(Hint: It's tajarans)